src/DDTrace/Integrations/SandboxedIntegration.php

```diff
@@ -21,4 +21,30 @@ public function addTraceAnalyticsIfEnabled(SpanData $span)
         }
         $span->metrics[Tag::ANALYTICS_KEY] = $this->configuration->getTraceAnalyticsSampleRate();
     }
+
+    /**
+     * Sets common error tags for an exception.
+     *
+     * @param SpanData $span
+     * @param \Throwable $throwable
+     */
+    public function setError(SpanData $span, \Throwable $throwable)
```

Comment:
Is this PHP 7 only? On PHP 5, the root exception type is "Exception", not "Throwable".

Reply:
Nope, very good point!